packages/server/src/routes/me/mint.ts

```diff
@@ -99,6 +102,19 @@ async function _mintInvitedBonus(
   }
   const allowedAmount = Config.getInvitedBonus(createdAt.toMillis());
   throwIfTooLarge(amount, allowedAmount);
+
+  // Verify that bonus hasn't been claimed.
+  if (
+    !(await transaction.get(
+      operations
+        .where("op_code", "==", OperationType.MINT)
+        .where("data.type", "==", MintType.INVITED_BONUS)
+        .where("creator_uid", "==", loggedInMember.id)
+    )).empty
+  ) {
+    throw new AlreadyMintedError(loggedInMember.id);
```

Comment:
Should we another error for this? I think AlreadyMintedError only refers to the referral bonus right now

Reply:
Okkkk ... not a fan of all these Error files (seems like a simple enum would suffice) but I guess we should be consistent